db/migrate/20180416124217_create_hw_infos.rb

```diff
@@ -12,6 +12,6 @@ def change
     end
 
     add_index :hw_infos, :system_id, unique: true
-    add_index :hw_infos, :uuid, unique: true
+    add_index :hw_infos, :uuid
```

Comment:
I don't think we need the index at all (nobody's going to query the DB with UUID condition).